content/docs/how-we-work/baseline/_index.md

```diff
@@ -191,6 +191,81 @@ Follow these steps only if the project doesn't already exist.
+
+   * **Notification Settings**: Customize how and where you receive alerts. Options include email, SMS, Slack, or other third-party integrations. Make sure to configure these settings to ensure timely notifications.
+   * **Alert Thresholds**: Define thresholds for different alert types. For example, you might set a threshold to alert you only if an error occurs more than 10 times in an hour, helping to avoid alert fatigue.
+
+3. Configure Slack Notifications:
+   * **Integrate Sentry with Slack**:
+     * Navigate to **Project Settings** > **Integrations**.
+     * Select **Slack** and follow the prompts to authenticate and authorize Sentry to access your Slack workspace.
+     * Choose the Slack channel where you want to receive alerts.
+   * **Set Up Slack Notification Rules**:
+     * In **Project Settings** > **Alerts**, create a new alert rule.
+     * Define the conditions for the alert, such as error type and frequency.
+     * Under **Actions**, select **Send a notification to Slack** and choose the integrated Slack channel.
+     * Save the rule to start receiving Slack notifications for the defined alerts.
+
+#### Creating Tickets from Alerts
+
+1. Review the Alert Details:
+   * Click on an alert to view detailed information, including stack traces, affected users, and the frequency of occurrences. Detailed error context helps in understanding the issue better and aids in faster resolution.
+   * **Error Context**: Review the contextual data provided by Sentry, such as the environment (production, staging), user actions leading to the error, and system state. This information is critical for diagnosing and resolving issues efficiently.
+
+2. Integrate Sentry with your ticketing system: (e.g. Jira, Github Issues).
+   * Navigate to **Project Settings** > **Integrations**.
+   * Select **Jira** and follow the prompts to authenticate and authorize Sentry to access your Jira instance.
+   * Configure the integration settings, such as default project and issue type, to streamline ticket creation.
+
+3. Create a Ticket:
+   * **Manually Creating a Ticket**:
+     * Click on the **Create Issue** button in the alert details view.
+     * Select **Jira** as the integration.
+     * Fill in the ticket details, including the summary, description, and priority. Include all relevant information provided by Sentry to ensure the issue is well-documented.
+     * Submit the ticket, which will then appear in your Jira project.
+   * **Automatically Creating Tickets**:
+     * Configure Sentry to automatically create Jira tickets for certain types of alerts. This can be done by setting up alert rules to trigger ticket creation.
+     * In **Project Settings** > **Alerts**, create a new alert rule and specify the conditions (e.g., error type, frequency) and actions (create a Jira ticket).
+     * Define the default project, issue type, and other ticket fields to ensure consistency and streamline the process.
```

Comment:
These are for two different audiences and it could end up confusing them. Point 2 is a one-time activity and there may be no alerts at that time. This is part of the set up for Sentry integration. The remaining points are something that a team member may do routinely when monitoring the project. Let's split it up along those lines: Sentry Setup and Routine Monitoring.

PS Setting up alerts in the previous section is also a one-time set up activity.